pages/1.11/installing/troubleshooting/index.md

```diff
@@ -2,19 +2,16 @@
 layout: layout.pug
 navigationTitle:  Troubleshooting
 title: Troubleshooting
-menuWeight: 400
-
+menuWeight: 40
 excerpt: Troubleshooting DC/OS installation issues
-
-enterprise: false
 ---
 
 # <a name="general"></a>General troubleshooting approach
 
-1.  Verify that you have a valid IP detect﻿⁠⁠⁠⁠ script, functioning DNS resolvers to bind the DC/OS services to, and that all nodes are synchronized with NTP.
+* Verify that you have a valid IP detect﻿⁠⁠⁠⁠ script, functioning DNS resolvers to bind the DC/OS services to, and that all nodes are synchronized with NTP.
 
 
-## <a name="ip-detect-script"></a>IP detect script
+## IP detect script
 
 You must have a valid [ip-detect](/1.11/installing/ent/custom/advanced/) script. You can manually run `ip-detect` on all the nodes in your cluster or check `/opt/mesosphere/bin/detect_ip` on an existing installation to ensure that it returns a valid IP address. A valid IP address does not have:
```

Comment:
Doesn't link to ip-detect script. Should link to: `/1.11/installing/production/advanced/#create-an-ip-detection-script`